jp2_online/templates/base/home.html

```diff
@@ -15,15 +15,15 @@
 
 <div class='row text-center'>
 	{% if user and not user.is_anonymous %}
-	<a>Hello, {{ user.get_full_name }}!</a>
+	<a><h4>Hello, {{ user.get_full_name }}!</h4></a>
 	<br>
 	<a href="/tosp_auth/logout">Logout</a>
 	{% else %}
-	Social login used to be here.
+	<a href="/tosp_auth/login">Inciar Sesión</a>
```

Comment:
Sure, I think this view is very useful, we can put the login form in a modal, but if they introduce wrong credentials, the easiest thing to do is to redirect them to this view, and also simplifies the presentation of error messages.

I'll fix the typo